client/src/components/SocialShare.tsx

```diff
@@ -46,6 +45,37 @@ type SocialShareProps = {
   customContent?: SocialShareContent;
 };
 
+const _FacebookButton: React.FC = (props) => (
```

Comment:
is the leading underscore meant to denote that this is a private variable? it's a convention I haven't seen in other places of our code but perhaps this is best practice we should start doing